### PR TITLE
Fixing some of the translation.id.json

### PR DESCRIPTION
**Pull request by kadektop2**
Readjusting some words to make it more sensible.

### CODE REVIEW

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json [hunk 1]
```diff
@@ -18,19 +18,19 @@
     },
     {
       "Key": "wrongAuthType",
-      "Value": "Jenis AuthType pada config.json tidak diketahui"
+      "Value": "Jenis AuthType pada config.json tidak dikenal"
     },
 	{
       "Key": "loginInvalid",
-      "Value": "Kredensial (nama pengguna dan kata kunci) cacat dan gagal masuk"
+      "Value": "Username dan password tidak cocok, login gagal!"
```
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: `login` is not a word in Indonesian.

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json [hunk 2]
```diff
@@ -330,15 +330,15 @@
     },
     {
       "Key": "recyclingQuietly",
-	  "Value": "Daur ulang senyap..."
+	  "Value": "Mendaur-ulang senyap..."
```
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Use `mendaur ulang` instead. (see: http://kbbi.web.id/daur%20ulang)

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json (hunk 1 above)
Comment (kadektop2) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Yes it is not in bahasa but it is kinda hard (and sounds lame) to put everything into bahasa.

Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: okay...

**Pull request merged**